### DISCUSSION TITLE
allow cline to find for files in entire project, just by exact or similar file name

### DISCUSSION THREAD

**Discussion by u007** [cline/cline#661] [Feature Requests]
need a way for cline todo its own file search, with given root path from chat, so we dont be limited by the file browsing capabilities which is impaired by depth level. for initial stage just use exact partial file name match via find method, then ask user to check 1 or more files

**Comment by u007** [comment]
i actually find bolt.new brilliantly works without much prompt, not sure is it due to context cache

**Comment by u007** [comment]
i would love to help, but can someone give me hint where todo this

**Comment by decentralize-all** [comment]
Search should just be done using VScode % search then it can provide a list of exact files as via a GenAI agent tool. Sending an entire codebase wont work.
